test-utils/contracts/alice/src/lib.rs

```diff
@@ -31,164 +26,159 @@ impl Alice {
     }
 }
 
-#[cfg(target_arch = "wasm32")]
-mod hosted {
+#[cfg(target_family = "wasm")]
+#[path = ""]
+mod wasm {
```

Comment:
Maybe do the same here and introduce an `alice-types` crate to structure this better?